dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -740,7 +735,8 @@ class DevtoolsStartupTest {
       unawaited(process.exitCode.whenComplete(() {
         didExit = true;
       }));
-      await Future.any(<Future<void>>[completer.future, Future<void>.delayed(const Duration(minutes: 5)), process.exitCode]);
+      await Future.any(
+          <Future<void>>[completer.future, Future<void>.delayed(const Duration(minutes: 5)), process.exitCode]);
```

Comment:
Ditto.

Reply:
Reverted back to previous formatting to simplify the code review